cmd/loop/loopout.go

```diff
@@ -235,6 +243,15 @@ func loopOut(ctx *cli.Context) error {
 		}
 	}
 
+	paymentTimeout := 0
+	if ctx.IsSet("payment_timeout") {
+		paymentTimeout = int(ctx.Duration("payment_timeout").Seconds())
```

Comment:
I propose to check that `paymentTimeout < math.MaxUint32` just in case, to make sure that it does not overflow in the conversion to `uint32` below. It should be always satisfied, because time.Duration does not support such large values.